src/common/packager.ts

```diff
@@ -182,4 +185,8 @@ export class Packager {
             return Q.resolve<void>(void 0);
         });
     }
+
+    private openFileAtLocationCommand(): string {
+        return path.join(__dirname, "..", "..", "atom");
```

Comment:
I'd suggest moving this `atom` script into a `/scripts` folder or something rather than just in the root of the package.